src/components/Sidebar/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { SidebarItem } from './components/SidebarItem';
+import './styles.scss';
+
+const USERS_WORKSPACES = [
+  {
+    title: 'My Personal Project',
+  },
+  {
+    title: 'Crop Monitoring',
+  },
+];
+
+const SIDEBAR_OPTIONS = [
+  {
+    title: 'Credentials',
+    link: '/credentials',
+  },
+  {
+    title: 'Workspaces',
+    link: '/workspaces',
+  },
+];
+
+type Sidebar = {
+  activeContent: string;
+  setActiveContent: (content: string) => void;
+};
+
+export const Sidebar = ({ activeContent, setActiveContent }: Sidebar) => {
+  return (
+    <div className="workspace__sidebar">
+      <h2>My Workspace</h2>
+
+      {USERS_WORKSPACES.map((workspace) => (
+        <div key={workspace.title} className="workspace__sidebar-group">
```

Comment:
Have you considered being more hierarchical in you SCSS structuring? e.g. duplication of `workspace__sidebar` is unnecessary, if you replace

```scss
.workspace__sidebar {
...
}

.workspace__sidebar-group {
...
}
```

with

```scss
.sidebar {
  .group {
    ...
  }
}
```

That should achieve the same thing and indicates to uses that the `group` is only ever used inside the `sidebar`, instead of prefixing the class names

Reply:
Both are fine with me, I've always followed using `__` and `-` and then use scss to do 
```
.workspace {
    &__sidebar {
        &-group {}
    }
}
```